src/Avalonia.Controls/Avalonia.Controls.csproj

```diff
@@ -26,6 +26,7 @@
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
   </PropertyGroup>
   <ItemGroup>
+    <Compile Include="..\Avalonia.Visuals\Utilities\StringTokenizer.cs" Link="Utils\StringTokenizer.cs" />
```

Comment:
I think I'd prefer to make `StringTokenizer` public rather than include it separately in different projects, it seems rather hackish like this? It should probably be moved into `Avalonia.Base\Utilities`.